Modules/posixmodule.c

```diff
@@ -900,6 +900,10 @@ typedef struct {
 static void
 path_cleanup(path_t *path)
 {
+#if !USE_UNICODE_WCHAR_CACHE
+    PyMem_Free((wchar_t *)path->wide);
```

Comment:
Should this maybe clear the field before freeing it? (Thinking of any interaction with tracemalloc or things like that?)

Reply:
I am not sure this is possible, but I will add a code just for the case.